Lib/test/test_asyncio/test_futures.py

```diff
@@ -949,13 +1032,19 @@ class CSubFuture(futures._CFuture):
             pass
         return CSubFuture(loop=self.loop)
 
+    def test_schedule_callbacks_list_mutation_3(self):
+        super()._test_schedule_callbacks_list_mutation_3(RuntimeError, 'corrupted')
```

Comment:
This could be the full RuntimeError message, eg "corrupted callbacks list" or "corrupted callback tuple"